src/redux/notifications/actions.ts

```diff
@@ -1,3 +1,267 @@
-// actions consts
-export const READ_NOTIFICATION = 'READ_NOTIFICATION';
-// export const DELETE_NOTIFICATION = 'DELETE_NOTIFICATION';
+import { HttpClient } from '../../core/utils/http-client';
+import CONFIG from '../../config';
+import { Notifications } from '../../core/messaging/notifications/notifications';
+import { captureException as SentryCaptureException } from '@sentry/react-native';
+import { getSelectedWallet } from '../wallets/selectors';
+import { Dispatch } from 'react';
+import { IReduxState } from '../state';
+import { PushNotifTokenType } from '../../core/messaging/types';
+import { IAccountState } from '../wallets/state';
+import { getTokenConfig } from '../tokens/static-selectors';
+import { getWalletCredentialsKey } from '../../core/secure/keychain/keychain';
+import * as schnorr from '@zilliqa-js/crypto/dist/schnorr';
+import moment from 'moment';
+
+export const SET_UNSEEN_NOTIFICATIONS = 'SET_UNSEEN_NOTIFICATIONS';
+export const SET_NOTIFICATIONS = 'SET_NOTIFICATIONS';
+export const MARK_SEEN = 'MARK_SEEN';
+
+const setUnseenNotifications = (unseenNotifications: number) => {
+    return {
+        type: SET_UNSEEN_NOTIFICATIONS,
+        data: { unseenNotifications }
+    };
+};
+
+export const setNotifications = (notifications: any) => {
+    return {
+        type: SET_NOTIFICATIONS,
+        data: { notifications }
+    };
+};
+
+const getTimestamp = () => {
+    return moment().unix() * 1000;
+};
+
+const getDomain = () => {
```

Comment:
we might use this for other functions in api, make an utils where the api is implemented